deployment/cfn/vpc.py

```diff
@@ -107,7 +107,7 @@ def set_up_stack(self):
             'BastionHostAMI', Type='String', Description='Bastion host AMI'
         ), 'BastionHostAMI')
 
-        self.create_vpc()
+        public_route_table = self.create_vpc()
 
         # TODO: Setup VPC endpoint to bypass NAT when talking to S3
```

Comment:
Addressed in https://github.com/WikiWatershed/model-my-watershed/commit/26133d4c0b41e14cd90b390d674d404491c7d15d.